doczrc.js

```diff
@@ -19,6 +19,7 @@ export default {
   menu: [
     { name: 'Introduction' },
     { name: 'Usage', menu: ['Basic Hooks', 'Advanced Hooks'] },
+    { name: 'Renderer' },
```

Comment:
I wasn't sure where to put this, but I felt this probably was a good place...

Reply:
Part of me wonders if this and the [Installation instructions](https://react-hooks-testing-library.com/#installation) section should be combined into an `Installation` page?

Reply:
That sounds like a good idea, i've probably repeated myself across in `Peer Dependencies` & `Renderer` anyway.